crates/fluvio-connector-package/test-data/connectors/full-config.yaml

```diff
@@ -11,6 +11,7 @@ meta:
     compression: gzip
   consumer:
     partition: 10
+    max_bytes: "1 MB"
   secrets:
```

Comment:
Can we add test cases for just regular numbers? For example, this should work
```
consumer:
      max_bytes: 13933
```
```

Reply:
Ok, I will update then to use:  https://docs.rs/bytesize-serde/latest/bytesize_serde/ that seems to handle that case

Reply:
Addressed in https://github.com/infinyon/fluvio/pull/3209/commits/869d4ef8b0b8dac2543311fdde36a80b359c2aec (also added test there)